src/style/foundation/main.less

```diff
@@ -131,3 +96,38 @@
 @import '../modal/service-modal';
 @import '../modal/legal-hold-modal';
 @import '../modal/invite-modal';
+
+@import '../components/search-list';
+@import '../components/group-list';
+@import '../components/user-list';
+@import '../components/search-input';
+@import '../components/availability-state';
+@import '../components/image';
+@import '../components/context-menu';
+@import '../components/copy-to-clipboard';
+@import '../components/device-card';
+@import '../components/ephemeral-timer';
+@import '../components/message-timer-button';
+@import '../components/full-search';
+@import '../components/input-level';
+@import '../components/info-toggle';
+@import '../components/group-video-grid';
+@import '../components/loading-bar';
+@import '../components/panel-participant-details';
+@import '../components/enriched-fields';
+@import '../components/legal-hold-dot';
+@import '../components/asset/common/common.less';
+@import '../components/asset/asset-header';
+@import '../components/asset/video-asset';
+@import '../components/asset/file-asset';
+@import '../components/asset/audio-asset';
+@import '../components/asset/link-preview-asset';
+@import '../components/asset/location-asset';
+@import '../components/asset/message-button';
+@import '../components/asset/controls/seek-bar';
+@import '../components/asset/controls/audio-seek-bar';
+@import '../components/asset/controls/media-button';
+@import '../components/calling/choose-screen';
+@import '../components/list/participant-item';
+@import '../components/mention-suggestions';
+@import '../components/list/conversation-list-calling-cell';
```

Comment:
Moving the components at the bottom of the file allows cascading to work as expected. The way it was set up before is probably why there's so many !important rules in the codebase.